docs/src/main/paradox/maven.md

```diff
@@ -2,4 +2,95 @@
 
 To get started with Akka gRPC read the [client](client.md) or [server](server.md) introductions.
 
-TODO document more advanced Maven plugin features here
\ No newline at end of file
+## Configuring what to generate
+
+The plugin can be configured to generate either Java or Scala classes, and then server and or client for the chosen language.
+By default both client and server in Java are generated.
+
+```xml
+<plugin>
+    <groupId>com.lightbend.akka.grpc</groupId>
+    <artifactId>akka-grpc-maven-plugin</artifactId>
+    <version>${akka.grpc.version}</version>
+    <configuration>
+      <language>JAVA</language> <!- or SCALA, must be uppercase -->
```

Comment:
can we fix that instead, and make it case insensitive?

Reply:
Agreed, #241